js/start.js

```diff
@@ -305,44 +346,63 @@ function fetchStartupData() {
           walletBalanceFetchFailed = true;
         } else if (jqXhr === searchProvidersFetch) {
           searchProvidersFetchFailed = true;
+        } else if (jqXhr === verifiedModsFetch) {
+          verifiedModsFetchFailed = true;
         }
       }
     })
     .done(() => {
       fetchStartupDataDeferred.resolve();
+      setInterval(() => fetchVerifiedMods(), 360000);
     })
     .fail((jqXhr) => {
       const curConn = getCurrentConnection();
 
-      if (!jqXhr || !curConn || curConn.status !== 'connected') {
+      // Check to see if the server is not connected. the search providers and verified moderators
+      // can return an undefined jqXhr if their external endpoints are down while the app itself is
+      // still functional.
+      if ((ownFollowingFailed || walletBalanceFetchFailed) &&
```

Comment:
If the search providers fail to load from localStorage or the verified moderators fail to load from an external source, this logic shouldn't trigger, since the node might be running normally and can proceed without the search providers or the verified moderators.